docs/guides/nginx/nginx-ssl.md

```diff
@@ -3,27 +3,30 @@ sidebar_position: 2
 ---
 
 # NGINX Proxy with SSL
-This section requires [NGINX](https://nginx.org/).
+This section requires valid installation of [NGINX](https://nginx.org/).
 
-## Cloudflare
+## Options:
+
+There exists several ways of handling SSL certificates. In this guide you can see how to do it with Cloudflare and certbot.
+
+### Cloudflare
 If using Cloudflare, you can generate a certificate for your domain through the dashboard. 
 
 First head over to your Cloudflare dashboard and select your domain. Then click the SSL/TLS tab and click "Client Certificates".
 
 Click "Create Certificate", and choose the following options.
 ![](/guides/cf-ssl-tls-creation.png)
 
-Click create once you have selected the options, and you will be given a certificate in the "PEM" format. Copy the certificates content and paste it into a file called `/etc/ssl/certs/<your domain>.key`.
+Click create once you have selected the options, and you will be given a certificate in the "PEM" format. Copy the certificates content and paste it into a file called `/etc/ssl/certs/<your domain>.pem`.
+
+Copy the private key's content and paste it into a file called `/etc/ssl/certs/<your domain>.key`.
 
-Copy the private key's content and paste it into a file called `/etc/ssl/certs/<your domain>.pem`.
+If your domain was `domain.xyz` you would have the following 2 files:
 
-If your domain was `diced.tech` you would have the following 2 files:
-```
-/etc/ssl/certs/diced.tech.key
-/etc/ssl/certs/diced.tech.pem
-```
+```/etc/ssl/certs/domain.xyz.pem``` with the certificate and
+```/etc/ssl/certs/domain.xyz.key``` with the private key.
```

Comment:
```suggestion
`/etc/ssl/certs/domain.xyz.key` with the private key.
```

Reply:
Inline code-blocks should use one back tick, instead of three even though they produce the same result.